Libraries/LibJS/Tests/exception-ReferenceError.js

```diff
@@ -1,9 +1,3 @@
-load("test-common.js");
-
-try {
-    i < 3;
-} catch (e) {
-    assert(e.name === "ReferenceError");
-}
-
-console.log("PASS");
+test("unknown variable produces ReferenceError", () => {
+    expect(new Function("i < 3")).toThrow(ReferenceError);
```

Comment:
Umm, if that's throwing that's a bug - just constructing the function object should not throw yet, only calling it should.

I.e.

```suggestion
    expect(new Function("i < 3")()).toThrow(ReferenceError);
```

(or rather the shortcut proposed above :) )

Reply:
`toThrow` operates on a function target, and does the calling itself :) however, I wrote this really weird, so I understand the confusion. It should be `expect(() => { i < 3; }).toThrow(ReferenceError)`.

Reply:
oh I see, what I proposed doesn't really make sense lol - my bad :D